model/vm_host.rb

```diff
@@ -178,4 +178,12 @@ def hetznerify(server_id)
       create_addresses
     end
   end
+
+  def reset
+    if Config.production?
+      fail "BUG: reset is only allowed in development"
+    end
```

Comment:
nit: if we only support it for development
```suggestion
    unless Config.development?
      fail "reset is only allowed in development"
    end
```